content/applications/general/voip/axivox/dial_plan_advanced.rst

```diff
@@ -0,0 +1,394 @@
+person or voicemail, this reduces wasted time and hangups.
+
+.. image:: dial_plan_advanced/time-condition.png
+   :align: center
+   :alt: Time condition element set in a dial plan on Axivox. Time condition is highlighted.
+
+.. important::
+   To set the :guilabel:`Timezone` that the :guilabel:`Time Condition` operates under, navigate to
+   `Axivox management console <https://manage.axivox.com>`_, and click :guilabel:`Settings` in the
+   menu on the left. Then, set the :guilabel:`Timezone` using the second field from the bottom by
+   clicking on the drop-down menu.
+
+Access list scenario
+--------------------
+
+An :guilabel:`Access List` element in a dial plan allows for the routing of certain numbers, and
+disallows (denies) other numbers.
+
+After adding an :guilabel:`Access List` element to a dial plan, it can be configured by
+double-clicking on the element directly in the :guilabel:`Dialplan Editor` pop-up window.
+
+Two fields appear where regular expressions can based in the :guilabel:`Allow` and :guilabel:`Deny`
+fields of the :guilabel:`Access List` pop-up window.
+
+.. example::
+   For a very important customer, their number can be set in the :guilabel:`Allow` field, and these
+   callers can be sent directly to management.
+
+.. tip::
+   A regular expression (shortened to regex or regexp), sometimes also referred to as a rational
+   expression, is a sequence of characters that specifies a match pattern in text.
+
+.. image:: dial_plan_advanced/access-config.png
+   :align: center
+   :alt: Access list element configuration with the allow/deny fields highlighted.
+
+.. example::
+   - `2\\d\\d`: validates numbers from 200 to 299
+   - `02\\d*`: validates all numbers beginning with 02
+   - `0017165551212`: validates the number (0017165551212)
+
+After setting the :guilabel:`Allow` and :guilabel:`Deny` fields with regular expressions or numbers,
+click :guilabel:`Save` on the :guilabel:`Access List` pop-up window.
+
+Then, on the :guilabel:`Access list` element in the dial plan, three paths (or routes) are available
+to link to further actions.
+
+Unknown calls can be routed through the regular menu flow by adding a :guilabel:`Menu` element, and
+connecting it to the :guilabel:`Unknown` path. :guilabel:`Refused` calls can be routed to the
+:guilabel:`Hang up` element. Lastly, :guilabel:`Authorized` callers can be sent to a specific
+extension or queue.
+
+.. image:: dial_plan_advanced/access-list.png
+   :align: center
+   :alt: Access list element highlighted in an example dial plan.
+
+Switches
+========
+
+A :guilabel:`Switch` element in Axivox is a simple activated/deactivated route action.
+
+These can be activated or chosen quickly, allowing for quick routing changes, without altering the
+dial plan.
+
+Alternate routes can be set to jump to, at a moment's notice, based on variables, such as
+availability, or to adjust traffic flow for any number of reasons.
```

Comment:
Alternate routes can be configured, so that in a moments notice, they can be switched to. This could
be for new availability, or to adjust traffic flow for any number of reasons.